tools/slack_tools/collect_prs.py

```diff
@@ -0,0 +1,192 @@
+# Script for collecting PRs in need of review, and informing maintainers via
```

Comment:
It's up to you, but my preference is to just run this as a daily GH action cron. It looks like it should be pretty easy to use an action with code that exists in the repo: https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions?learn=getting_started#referencing-an-action-in-the-same-repository-where-a-workflow-file-uses-the-action.

Do you want to do this as a follow up or try to get this working now? I can help offline with credentials, etc.

Reply:
I think follow-up if that's OK

Reply:
Sure that's fine. Per offline convo I will help set up a GH action template that we can move this to in parallel.